### PR TITLE
add partisan affiliation data to graph

### PR DESCRIPTION
**Pull request by aristeinberg**
This adds a checkbox below the main graph which toggles the graph into a partisan mode. See screenshots:

image

image

### CODE REVIEW

File: db/load_history.sql [hunk 1]
```diff
@@ -0,0 +1,21 @@
+-- download 2020.zip from 
+-- https://elections.sos.ga.gov/Elections/voterhistory.do
+-- unzip it and then run this script.
```
Comment (aristeinberg) [inline db/load_history.sql:3]: i suppose we might want to add some more instructions to the readme and maybe something in the Makefile to give info about how to do this...welcome suggestions about what to put in Makefile

File: static/homepage.js [hunk 2]
```diff
@@ -29,10 +61,18 @@
       y: {
         label: "Votes Cast",
         min: 0,
-        max: 4000000,
```
Comment (aristeinberg) [inline static/homepage.js:32]: dropping this line does prevent us from seeing the "high water mark", but the upside is that it's a lot easier to see the detail on the D vs R graph when it's zoomed in.

File: templates/base.html [hunk 3]
```diff
@@ -47,7 +47,7 @@
     integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2"
     crossorigin="anonymous">
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-  <link href="/static/styles.css" rel="stylesheet">
+  <link href="/static/styles.css?1" rel="stylesheet">
```
Comment (aristeinberg) [inline templates/base.html:50]: this (and the similar line on the js file) is obviously not the right way to do things. do we need to add some kind of cache-control to our static directory? not totally sure how to do that... i did need to use shift+reload while i was debugging in order to get my code to work.

File: static/homepage.js (hunk 2 above)
Comment (aristeinberg) [reply static/homepage.js:32]: (i now fixed this by leaving the max in place when not viewing the partisan split)

File: templates/base.html (hunk 3 above)
Comment (benweissmann) [reply templates/base.html:50]: You can disable the cache when the dev tools are open -- there's a checkbox under "Network" in the "Preferences" page of the "Settings" UI in the dev tools (hit the gear icon):

Screen Shot 2020-12-22 at 11 40 41 AM

**Comment by benweissmann** [comment]
Sorry for the delay, this looks great!

File: templates/base.html (hunk 3 above)
Comment (aristeinberg) [reply templates/base.html:50]: Yeah I was not concerned about my dev environment. Rather I was concerned about deploying new code and having a viewer display it with cached static files.

it's actually even worse than i thought b/c of the cloudflare caching - if we deploy new static files without modifying the cachebreaker here, then everyone (even people who never previously saw the site) will continue to get served the old version of the static files until cloudflare invalidates its cache.

given we've just got two weeks til the election, i'm not sure it's totally necessary to fix this, but i think in an ideal world we'd probably have some kind of build process that calculates a hash of the static file contents and appends it to the url here?

Comment (benweissmann) [reply templates/base.html:50]: The cloudflare cache is 30 minutes and the [in-browser cache is 15 minutes](https://github.com/vote/gaballot/blob/main/app.py#L70) so i'm not too worried. We can also manually invalidate the cloudflare cache from the cloudflare console. If we want to, we could look into something like this to automatically invalidate the cache on deploy: https://elements.heroku.com/buildpacks/jamesmartin/cloudflare-cache-purge-buildpack

**Pull request merged**